src/vdaf/prio3.rs

```diff
@@ -281,8 +278,9 @@ where
         measurement: &T::Measurement,
         rand_source: RandSource,
     ) -> Result<Vec<Prio3InputShare<T::Field, L>>, VdafError> {
```

Comment:
This would add some much needed clarity I think (maybe make declare it above for the whole `vdaf` module)
```suggestion
    ) -> Result<Vec<Prio3InputShare<T::Field, L>>, VdafError> {
    const DST_LEN: usize = VERSION.len() + 4;
```

Reply:
Yeah, or write this in terms of `Self::ID.to_be_bytes().len()` to absolve my rotten decrepit brain from figuring out what 4 and 5 refer to.